integration/commands_freemonitoring_test.go

```diff
@@ -64,24 +57,20 @@ func TestCommandsFreeMonitoringSetFreeMonitoring(t *testing.T) {
 	})
 
 	for name, tc := range map[string]struct {
-		command bson.D
-		err     *mongo.CommandError
+		command        bson.D
+		err            *mongo.CommandError
+		expectedRes    bson.D
+		expectedStatus string
 	}{
 		"Enable": {
-			command: bson.D{{"setFreeMonitoring", 1}, {"action", "enable"}},
```

Comment:
Where do we plan to cover these tests where using command is disabled?

Reply:
I don't think there's easy way to make a check for that. We would need to initialize another FerretDB instance for that and we don't do anytthing similar in any test. I took the advice to make tests simple and get some feedback so if needed we would make some changes in tests

Reply:
That's right. We run integration tests against an already running ferretdb, so it won't be simple. For now, let's go with what we have :)